src/renderer/components/Passphrase/PassphraseInput.vue

```diff
@@ -29,6 +29,7 @@
         :title="$t(passphraseIsVisible ? 'PASSPHRASE_INPUT.HIDE' : 'PASSPHRASE_INPUT.SHOW')"
         class="PassphraseInput__visibility-button flex flex-no-shrink text-grey-dark hover:text-blue mr-2"
         type="button"
+        tabindex="-1"
```

Comment:
You will probably need to do the same with QR code too I think?

Reply:
yes, I've also tested that now you can't tab to QR buttons (including WalletHeadingPrimaryActions)

Reply:
There's the ButtonModal for QR code just below this [here](https://github.com/ArkEcosystem/desktop-wallet/pull/933/files#diff-e14ef7365d41282ea6f8dccfb7708b73R41) which doesn't seem to have the tabindex still. Selecting the PassphraseInput, doing 1 tab then `document.activeElement` in devtools shows as the QR code button is selected. Could you update that one also please